### PR TITLE
feat(activities): add REST services and pages to manage activity types for persons

### PR DESCRIPTION
**Pull request by jnizet**

### CODE REVIEW

**Comment by jnizet** [comment]
@acrepet Tests ae still missing for the frontend part, but you can test right away if you want to. I only list the basic information of persons in the "participants" pages (as in the person list page), but this could be changed easily if they want to see the phone number and/or email address of participants. But I don't remember if this is necessary in that page, or if it will be in the future concrete activity pages.

**Comment by acrepet** [comment]
Great! I will test it. And I think that basic informations in the list of
people are sufficient

Le 29 sept. 2017 7:13 PM, "JB Nizet" <notifications@github.com> a écrit :

@acrepet <https://github.com/acrepet> Tests ae still missing for the
frontend part, but you can test right away if you want to. I only list the
basic information of persons in the "participants" pages (as in the person
list page), but this could be changed easily if they want to see the phone
number and/or email address of participants. But I don't remember if this
is necessary in that page, or if it will be in the future concrete activity
pages.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/Ninja-Squad/globe42/pull/107#issuecomment-333184459>,
or mute the thread

**Comment by jnizet** [comment]
I've now added the tests. Ready to review, test, and merge if all goes well.

File: frontend/src/app/person-participations/person-participations.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<p id="message">{{ person | fullname }} participe aux types d'activité suivants&nbsp;:</p>
```
Comment (acrepet) [inline frontend/src/app/person-participations/person-participations.component.html]: aux types d'activité suivants -> aux types d'activitéS suivants

**Comment by acrepet** [comment]
Besides my nitpicking about singular/plural in a sentence... LGTM!

**Comment by jnizet** [comment]
It seems you're right (I would have bet the contrary): https://www.frantastique.com/fr/regles-orthographe/des-types-de-vetements-des-types-de-cuisson. I'll fix that.

**Pull request merged**